airbyte-scheduler/persistence/src/test/java/io/airbyte/scheduler/persistence/WorkspaceHelperTest.java

```diff
@@ -64,30 +62,38 @@
       .withName("dest")
       .withTombstone(false);
   private static final StandardSync CONNECTION = new StandardSync()
+      .withName("a name")
       .withConnectionId(CONNECTION_ID)
       .withSourceId(SOURCE_ID)
       .withDestinationId(DEST_ID).withCatalog(new ConfiguredAirbyteCatalog().withStreams(new ArrayList<>()))
       .withManual(true);
   private static final StandardSyncOperation OPERATION = new StandardSyncOperation()
       .withOperationId(OPERATION_ID)
       .withWorkspaceId(WORKSPACE_ID)
+      .withOperatorType(OperatorType.DBT)
       .withName("the new normal")
       .withOperatorNormalization(new OperatorNormalization().withOption(Option.BASIC))
       .withTombstone(false);
 
-  Path tmpDir;
   ConfigRepository configRepository;
   JobPersistence jobPersistence;
   WorkspaceHelper workspaceHelper;
   ConnectorSpecification emptyConnectorSpec;
 
   @BeforeEach
-  public void setup() throws IOException {
-    tmpDir = Files.createTempDirectory("workspace_helper_test_" + RandomStringUtils.randomAlphabetic(5));
-
-    configRepository = new ConfigRepository(new FileSystemConfigPersistence(tmpDir), new NoOpSecretsHydrator(), Optional.empty(), Optional.empty());
+  public void setup() throws IOException, JsonValidationException, ConfigNotFoundException {
```

Comment:
:information_source: This tests where still using the `FileSystemConfigPersistence` to actually be able to write something out and persist it during a test. Since all tests are just testing for a proper cached implementation, I've instead replaced this by more explicitly checking on the mock calls.